bionty/base/entities/_protein.py

```diff
@@ -26,4 +26,6 @@ def __init__(
         version: Literal["2023-02", "2023-03", "2024-03"] | None = None,
         **kwargs,
     ) -> None:
-        super().__init__(source=source, version=version, organism=organism, **kwargs)
+        super().__init__(
+            source=source, version=version, organism=organism, has_ols=False, **kwargs
```

Comment:
Consider a clearer name than `has_ols` and documenting it? I'd not understand what this means looking at the code.

Reply:
`is_on_ols`?

`obtainable_via_ols`?